variables.ini.dist

```diff
@@ -1,147 +1,142 @@
 [COMMON]
-; Examples of each are listed behind the ;
-SERVER_URL = ; https://www.my-website.com
-LDAP_SERVER = ; ldap://my.ldap.com
-LDAP_SERVER_DN = ; ou=example,dc=org
-TOKEN_EXPIRY_TIME_DAYS = ; 1
-DJANGO_SERVER_URL = ; https://www.my-website.com
-VIRTUALENV_PATH =; /path/virtualenv/atmo
+# Examples of each are listed behind the #
+SERVER_URL = # https://www.my-website.com
+LDAP_SERVER = # ldap://my.ldap.com
+LDAP_SERVER_DN = # ou=example,dc=org
+TOKEN_EXPIRY_TIME_DAYS = 1 # 
```

Comment:
Yes, I am trying to see how it would affect the `token_expiry_time`. 1 day is too long for testing but currently I have to work on something else. So, will test it with a shorter time later on. Also, for generating `local.py`, this is an essential variable that needs to have at least a value there